packages/material-ui-codemod/README.md

```diff
@@ -21,19 +21,220 @@ APIs.
 
 ### v5.0.0
 
-#### `box-sx-prop`
+#### `component-rename-prop`
+
+A generic codemod to rename any component prop.
+
+```diff
+-<Component prop="value" />
+-<Component prop />
++<Component newProp="value" />
++<Component newProp />
+```
+
+```sh
+find src -name '*.js' -print | xargs npx jscodeshift -t ../../material-ui/packages/material-ui-codemod/src/v5.0.0/component-rename-prop.js --component=Grid --from=prop --to=newProp
+```
+
+#### `autocomplete-rename-closeicon`
+
+Renames `fade` style utility import and calls frpm `fade` to `alpha`.
+
+```diff
+-<Grid justify="left">Item</Grid>
++<Grid item justifyContent="left">Item</Grid>
+```
+
+```sh
+find src -name '*.js' -print | xargs npx jscodeshift -t node_modules/@material-ui/codemod/v5.0.0/autocomplete-rename-closeicon.js
+```
+
+#### `avatar-circle-circular`
+
+Updates the Avatar `variant` value and classes key from 'circle' to 'circular'.
+
+```diff
+-<Avatar variant="circle" />
+-<Avatar classes={{ circle: 'className' }} />
++<Avatar variant="circular" />
++<Avatar classes={{ circular: 'className' }} />
+```
+
+```sh
+find src -name '*.js' -print | xargs npx jscodeshift -t node_modules/@material-ui/codemod/v5.0.0/avatar-circle-circular.js
+```
+
+#### `box-borderradius-values`
 
 Updates the Box API from separate system props to `sx`.
 
-The diff should look like this:
+```diff
+-<Box borderRadius="borderRadius">
+-<Box borderRadius={16}>
+-<Box borderRadius={1}>
+-<Box borderRadius="16px">
+```
+
+```sh
+find src -name '*.js' -print | xargs npx jscodeshift -t node_modules/@material-ui/codemod/v5.0.0/box-borderradius-values.js
+```
+
+#### `button-color-prop`
+
+Removes the Chip `variant` prop if the value is `"default"`.
 
 ```diff
--<Box border="1px dashed grey" p={[2, 3, 4]} m={2}>
-+<Box sx={{ border: "1px dashed grey", p: [2, 3, 4], m: 2 }}>
+-<Button color="primary">
+-<Button color="default">
++<Button>
++<Button>
 ```
 
 ```sh
+<<<<<<< HEAD
 npx jscodeshift --extensions js,ts,jsx,tsx --parser tsx -t node_modules/@material-ui/codemod/v5.0.0/box-sx-prop.js ./src
+=======
+find src -name '*.js' -print | xargs npx jscodeshift -t node_modules/@material-ui/codemod/v5.0.0/button-color-prop.js
+>>>>>>> [codemod] button-color-prop, chip-variant-prop
+```
```

Comment:
⚠️

Reply:
Oops!

Reply:
Should the former be preferred, since it covers both js & ts? 

If so, I'll update the rest to match

Reply:
Yes, the former is better.